frontend/src/modules/machinery/components/CaighdeanTranslation.js

```diff
@@ -0,0 +1,43 @@
+/* @flow */
+
+import React from 'react';
+import { Localized } from '@fluent/react';
+
+
+type Props = {|
+    source: {
+        type: string,
+        url: string,
+    },
+    index: number
+|};
+
+
+/**
+ * Show the translation source from Caighdean Machine Translation.
+ */
+export default class CaighdeanTranslation extends React.Component<Props> {
+    render() {
+        const {source, index} = this.props;
+
+        return <ul className="sources">
```

Comment:
These should return an `<li>` element, the `<ul>` should be set outside, in the `Translation` component.

Reply:
Okay!